src/ua/artcode/udiary/rest/AddRecordHandler.java

```diff
@@ -36,6 +36,10 @@ public void handle(HttpExchange httpExchange) throws IOException {
             //send success message
             Record saved = mainController.addRecord(record);
 
+            if (saved != null){
```

Comment:
if saved is null we will not show logs. Logs should notify us about not only specific cases. We should be informed about all situations(exceptions also)